### PR TITLE
New Opportunity StepZen

### PR DESCRIPTION
**Pull request by debxrshi**
Added StepZen

### CODE REVIEW

**Comment by debxrshi** [comment]
@monizb New PR

File: src/list.json [hunk 1]
```diff
@@ -251,6 +251,14 @@
         ],
         "description": "Contribute to ServiceNow to receive a free tshirt",
         "link": "https://developer.servicenow.com/blog.do?p=/post/hacktoberfest-2022/"
+      },
+      {
+        "organization": "StepZen",
```
Comment (thefool76) [inline src/list.json:256]: Please Re-Style it by using this command in the Repo Directory.
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2339735/patch | git am`
`git push`

**Review by thefool76 — CHANGES_REQUESTED** [review]

File: src/list.json (hunk 1 above)
Comment (debxrshi) [reply src/list.json:256]: Not working

Comment (debxrshi) [reply src/list.json:256]: by repo directory you mean the directory of the forked repo right?

Comment (debxrshi) [reply src/list.json:256]: Ran the commands and output returnd: Applying: Restyled by jq, prettier-json but still checks failing

Comment (debxrshi) [reply src/list.json:209]: Done @thefool76

**Review by thefool76 — APPROVED** [review]

**Comment by thefool76** [comment]
looks good! well done @debxrshi

**Pull request merged**